packages/ckeditor5-dev-tests/lib/utils/manual-tests/compilehtmlfiles.js

```diff
@@ -80,8 +92,41 @@ function uniq( arr ) {
 	return Array.from( new Set( arr ) );
 }
 
-function changeExtension( file, newExt ) {
+function setExtension( file, newExt ) {
 	const { dir, name } = path.parse( file );
 
 	return path.join( dir, name + '.' + newExt );
 }
+
+function getFilePathWithoutExtension( file ) {
+	const { dir, name } = path.parse( file );
+
+	return path.join( dir, name );
+}
+
+function watchFiles( filePaths, onChange ) {
+	for ( const filePath of filePaths ) {
+		watchFile( filePath, onChange );
+	}
+}
+
+function watchFile( filePath, onChange ) {
```

Comment:
This function is unnecessary. Don't split code into too many functions if not needed. I know this seems to be right to split code, but there are evidence that this reduces readability.